src/pseudopeople/loader.py

```diff
@@ -52,49 +53,59 @@ def load_and_prep_1040_data(data_path: dict, user_filters: List[Tuple]) -> pd.Da
     )
     # Rename tax_dependents columns
     dependents_wide = dependents_wide.add_prefix("dependent_").reset_index()
+    # Make sure we have all dependent columns if data does not have a guardian with 4 dependents
+    if COLUMNS.dependent_4_first_name.name not in dependents_wide.columns:
+        for column in [col.name for col in COLUMNS if "dependent_4" in col.name]:
+            dependents_wide[column] = np.nan
+
```

Comment:
This just adds dependent 4 metadata if we do not have a guardian with more than 3 dependents.